### PR TITLE
implemented cobinhood pairs, markets, orderbooks

### PR DESCRIPTION
**Pull request by mwlang**
- Introduces Cobinhood exchange with implementation of Market, Ticker, and Order book.
- [Y] I have added Specs
- [N] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [Y] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

I am not 100% sure how to confirm volume is indeed expressed in base.  However, I did confirm the Volume matches what's displayed on Cobinhood's exchange website.

### CODE REVIEW

**Comment by mwlang** [comment]
The CI failed with system errors where DNS for rubygems.org was not being resolved so nothing could be built.  If the build can be re-triggered, it should pass all specs.

File: lib/cryptoexchange/exchanges/cobinhood/services/pairs.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module Cryptoexchange::Exchanges
+  module Cobinhood
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Cobinhood::Market::API_URL}/market/trading_pairs"
+
+        def fetch_via_api(endpoint = self.class::PAIRS_URL)
+          ctx = OpenSSL::SSL::SSLContext.new
+          ctx.verify_mode = OpenSSL::SSL::VERIFY_NONE
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/cobinhood/services/pairs.rb]: Is it possible to make do without SSL verify none?

File: lib/cryptoexchange/exchanges/cobinhood/services/market.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Cobinhood::Market::NAME
+          ticker.last      = NumericHelper.to_d(market['last_trade_price'])
+          ticker.high      = NumericHelper.to_d(market['24h_high'])
+          ticker.low       = NumericHelper.to_d(market['24h_low'])
+          ticker.ask       = NumericHelper.to_d(market['lowest_ask'])
+          ticker.bid       = NumericHelper.to_d(market['highest_bid'])
+          ticker.volume    = NumericHelper.to_d(market['24h_volume'])
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/cobinhood/services/market.rb]: 👍

File: lib/cryptoexchange/exchanges/cobinhood/services/pairs.rb (hunk 1 above)
Comment (mwlang) [reply lib/cryptoexchange/exchanges/cobinhood/services/pairs.rb]: Right now, Cobinhood's SSL certificate is not valid.  The only way I found around it was to not verify it.  Perhaps merge for now, but keep an open issue ticket to check every 30 days or so so see if they've fixed their certificate issues?

Comment (tmlee) [reply lib/cryptoexchange/exchanges/cobinhood/services/pairs.rb]: @mwlang Tested on my local machine without SSL seem to work fine, i'll accept this PR and remove the verify none SSL and see how this goes moving forward

**Comment by tmlee** [comment]
:+1: Thank you!

**Pull request merged**